k4FWCore/scripts/k4run

```diff
@@ -60,14 +60,9 @@ class LoadFromFile(argparse.Action):
                 proptype = type(propvalue[0])
               propnargs = "+"
 
-          # add the argument twice, once as "--PodioOutput.filename"
-          # and once as "--filename.PodioOutput"
           propName = conf.name() + '.' + prop
           propNameReversed = prop + '.' + conf.name()
-          option_db[propName] = (conf, propName)
-          parser.add_argument( "--%s" % propName, "--%s" % propNameReversed, type=proptype, help=props[prop][1],
```

Comment:
Why do you remove this? This was added on purpose: Argparse autocompletes if possible, so adding `--filename.PodioOutput` allows using shorthand arguments like `--filename` (at least if filename is a unique property in the configurables